### PR TITLE
Home backend finished, frontend fixes and fixes

### LINKED ISSUE #17: Home-backend
# REQUIREMENTS
-Backend for home controller to let the admin login and log out
Implement the basic auth.
Create the sesions in the app.
Implementate in the admin model the encrypted password
Fix some front-end problems

### PR DESCRIPTION
**Pull request by javs1**
Added admin model with encrypted password using bcrypt gem
Added bcrypt gem to the app
Added login method to home controller
 - validate the user and create the session in case of succes
 - in case of error alert the user about a wrong user or password
Fixed home view frontend
 - in a merge by accident the frontend of the home page was modified,
   we fixed to the updated version and added a notice
 - modified form to send post request for login
Modified routes.rb
 - added route for post for login in home controller
Fixed database error
 - changed the name of employee migration to be created after branches
   migration
Created helper method
 - Method logged_in? that returns true if the session is created

Co-authored-by: Joshsam31 <samano82324@gmail.com>
Co-authored-by: javs1 <javierverduzco13@gmail.com>

# Description
What did you implemented/Why did you implemented this:
 - Explain the changes you’ve made.
 - It doesn’t need to be fancy and you don’t have to get to technical, yet.
 - Just explicit what you have implemented at a high level.
 - Let the reviewer know the overall effect of the PR.
 - Reference a ticket in your issue tracker. Explain what the change is and then and only then reference the ticket.
 - The “why” is a chance to explain both the engineering goal, but also a some business objective that is satisfied or moved along.

 Examples:
 - What?: I've added support for authentication to implement Key Result 2 of OKR1. It includes
model, table, controller and test. For more background, see ticket #JIRA-123.
 - Why?: These changes complete the user login and account creation experience. See #JIRA-123 for more information.

 # Testing
 Help me how can I test or look at the changes

 Example:
  - I've added coverage for testing all new methods. I used Faker for a few random user emails and names.

  ## Screenshots
If applicable, include screenshots of the results or screenshots that help to see changes

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1,59 +1,48 @@
-
 <html>
 
     <head>
 
     </head>
 
     <body>
```
Comment (YilKanda) [inline app/views/home/index.html.erb]: Why is this code in this file?

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -1,59 +1,48 @@
...
-<div></div>
     </body>
-
-
     <footer>
     </footer>
-
-
-</html>
+</html>
```
Comment (YilKanda) [inline app/views/home/index.html.erb]: Add a blank line.

File: app/models/admin.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Admin < ApplicationRecord
+  has_secure_password
+
+end
```
Comment (YilKanda) [inline app/models/admin.rb]: Add a blank line.

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -1,59 +1,48 @@
...
+			<div class="login-container">
+				<div class="d-flex justify-content-center h-100">
+					<div class="card">
+						<div class="card-header">
+							<h3>Sign In</h3>
+							<div class="d-flex justify-content-end social_icon">
+								<span><i class="fab fa-facebook-square"></i></span>
+								<span><i class="fab fa-google-plus-square"></i></span>
+								<span><i class="fab fa-twitter-square"></i></span>
+							</div>
```
Comment (YilKanda) [inline app/views/home/index.html.erb]: Check the indentation in your file.

File: app/views/home/index.html.erb (hunk 1 above)
Comment (javs1) [reply app/views/home/index.html.erb]: We didnt know what is this refering to, but we thought it was about the empty header and we deleted it

**Review by xFerchoVJ — COMMENTED** [review]
Everything looks good, just pay attention to the comments that gave you feedback!

**Review by xFerchoVJ — APPROVED** [review]

**Pull request merged**